block-processor/src/main/scala/net/santiment/btc/blockprocessor/RawBlockParserFlaMap.scala

```diff
@@ -0,0 +1,89 @@
+package net.santiment.btc.blockprocessor
+
+import com.typesafe.scalalogging.LazyLogging
+import net.santiment.btc.BitcoinClient
+import org.apache.flink.api.common.functions.RichFlatMapFunction
+import org.apache.flink.api.common.state.{ValueState, ValueStateDescriptor}
+import org.apache.flink.api.common.typeinfo.TypeInformation
+import org.apache.flink.configuration.Configuration
+import org.apache.flink.util.Collector
+import org.apache.flink.api.scala._
+
+import scala.collection.JavaConverters._
+
+class RawBlockParserFlaMap extends RichFlatMapFunction[RawBlock, UnmatchedTxEntry]
```

Comment:
There is a typo in the name of the class

```suggestion
class RawBlockParserFlatMap extends RichFlatMapFunction[RawBlock, UnmatchedTxEntry]
```